src/main/java/com/google/edith/servlets/DealsServlet.java

```diff
@@ -1,5 +1,6 @@
 package com.google.edith;
```

Comment:
ideally it would be package com.google.edith.servlets, but if I change it it will give me a 503 error because it thinks theres a servlet w the same name at com.google.edith and com.google.edith.packages. the only way i've found to fix that is to completely delete the project, restart the shell and re clone it. i was thinking maybe i could change it at the very end?